### DISCUSSION TITLE
Something changed and some elements are stuck light mode

### DISCUSSION THREAD

**Discussion by HiddenKnife526** [darkreader/darkreader#12577] [General]
Everything was fine today and Dark Reader was functioning perfectly. A little bit ago though, everything turned light mode again. I figured it was an update or something, and toggled Dark Reader off and on like I usually do when that happens. However, some elements aren't going dark like they are supposed to. Some drop-down menus and text entry fields are stuck with light backgrounds and light text. Others are worse and show light text on light backgrounds.
![Dark Reader issue](https://github.com/darkreader/darkreader/assets/142692361/45374a67-8054-465c-9cbd-ebb056b42345)
There is an image of what I mean on one of the sites I use. There is supposed to be readable text in the button and drop-downs along the bottom. and the big text field should be dark. I have tried using some of the other theme generation mods but that just made the problem worse or caused other issues on the affected sites.

I am using a fully updated Firefox on Windows 10 with the latest update on Dark Reader.

Is anyone else experiencing this issue, and know of a solution?

**Comment by Abkaal** [comment]
I also have this issue and no solution. Dark Reader used to nicely darken light debug toolbars but now it either ignores it or, when forced, turns everything to black and white rendering it useless as a debug tool.

Vivaldi (chromium) on Linux Mint here.

**Comment by HiddenKnife526** [comment]
The new update that came out today for Firefox fixed the problem. A big thanks to the people who work on the extension.
